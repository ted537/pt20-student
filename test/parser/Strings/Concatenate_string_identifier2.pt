using input;
var str = "Hello";
var str2 = "Hello";
var con = str | str2;

