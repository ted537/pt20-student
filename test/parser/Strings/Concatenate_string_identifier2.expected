 .sProgram
   .sIdentifier
  % .sNewLine
  .sParmEnd
  .sBegin
    .sVar
     .sIdentifier
     .sInitialValue
         .sStringLiteral
         .sExpnEnd
    % .sNewLine
    .sVar
     .sIdentifier
     .sInitialValue
         .sStringLiteral
         .sExpnEnd
    % .sNewLine
    .sVar
     .sIdentifier
     .sInitialValue
         .sIdentifier
         .sIdentifier
       .sConcatenate
       .sExpnEnd
    % .sNewLine
    % .sNewLine
  .sEnd
